### DISCUSSION TITLE
Year 2016 not working. roster attribute not present

### DISCUSSION THREAD

**Discussion by shibedroid** [cwendt94/espn-api#242] [ESPN API] [answered]
Hi, first of all thank you for all your incredible work! 

I am trying to compile some league stats across the seasons and for that I am importing the data from each season but I found that when importing specifically 2016 (my league has existed since 2013)  there is an error, every other season since 2013 worked fine.  I included an image:

![image](https://user-images.githubusercontent.com/90672001/133549765-d91621f5-141d-4d16-a8f7-a8c6ffec6f28.png)

it says "Key Error: 'roster'". Is this a common problem specific to the 2016 season? Is there any way to import that year?

Thank you very much!

**Comment by cwendt94** [comment, marked as answer]
Thats odd its only happening for the 2016 season. Did a team leave in the middle of the season or something special happen that season? I will look into this issue and see if it can be fixed!

**Reply by shibedroid** [reply]
Hi, still new to GitHub and I did not know you had replied

I checked and all teams are the same from 2015 and still the same in 2017, no movements at all, al team id the same. The info is in the espn page.

The only different thing I see on that season is that it was the last season the league used divisions, but I am not sure if I removed them after the draft, during the season or before beggining the 2017 season, I am not even sure if divisions can be removed during the season!, but in espn the season appears still with divisions.

Thanks for the help

**Reply by cwendt94** [reply]
So for 2016 all teams have a defined roster? Not sure what could be causing this, I checked my league that had a season in 2016 and no errors. What I can do, is add a fix so it won't throw an error but it seems like one or more teams won't have a roster

**Reply by cwendt94** [reply]
Try getting your 2016 season with the latest package version v0.18.3. This should at least stop it from erroring on the one team

**Reply by shibedroid** [reply]
thank you!! it worked!! I checked all the teams and there is something definitively weird there.

Team 3 had only its starters, no bench
Team 4 had a completely empty roster

Both teams had scores for the last week of the season, so both teams at that point had a roster, maybe an error on ESPN databases?

Anyway, it worked. Thank you!!!

**Reply by cwendt94** [reply]
Awesome I am glad you are able to get the 2016 season! Yeah that's definitely a possibility, very weird.
